src/deepsparse/v2/schedulers/continuous_batching_scheduler.py

```diff
@@ -28,11 +28,32 @@
 __all__ = ["ContinuousBatchingScheduler"]
 
 
+_GLOBAL_SCHEDULER = None
+
+
 class ContinuousBatchingScheduler(OperatorScheduler):
     """
     Manages EngineOperator jobs that should be run with continuous batching.
     Groups requests for the same engine into larger batches and returns
-    the result to the respeictive request threads after scheduled completion
+    the result to the respective request threads after scheduled completion
+
+    Example code for getting or creating a shared instance for scheduling
+    between pipelines and adding an engine operator to the scheduler
+    within a pipeline
+
+    ```python
+
+    class MyPipeline(Pipeline):
+
+        def __init__(self):
+            ...
+            engine_operator = EngineOperator(...)
+            ...
+            continuous_batching_scheduler = ContinuousBatchingScheduler.get_instance()
+            continuous_batching_scheduler.add_engine_operator(engine_operator)
+
```

Comment:
If this is the lifecycle we're proposing, doesnt that mean we can only start with one worker at any given point?

Reply:
yes, I think in general we will only want one worker so that for a given batch the engine will have full access to all resources. otherwise we'd need to run the engines in multistream mode which I am not sure is supported for KV cache right now